src/scripts/main.js

```diff
@@ -354,7 +354,27 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dash = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach((person) => {
+  const ageP = person.died - person.born;
+  const centuryP = Math.ceil(person.died / 100);
+  let gender = 'Female';
```

Comment:
The initial assignment of `gender` to 'Female' is correct given the logic, but it might be clearer to initialize it as `null` or an empty string and then set it based on the condition. This would make it explicit that the gender is determined by the condition.